.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: package-plugin-workflow
 
-on: push
+on: 
+  push:
+    paths-ignore:
+      - "**.md"
+      - "**ue-docker.yml"
```

Comment:
This is to be ignored? I'd assume that changes there would be the trigger for this workflow.

Reply:
There's no need to run CI checks when modifying the `ue-docker.yml` workflow as they are not affected by the result of Docker build. That's because Docker image build takes significantly longer than an average CI run and the new/updated images simply won't be available at that moment when they're consumed by `ci.yml`.